Assets/Scripts/Layers/Adapters/DataTypeAdapters/GeoJSONImportAdapter.cs

```diff
@@ -66,8 +70,14 @@ private void CreateGeoJSONLayer(LocalFile localFile, UnityEvent<string> onErrorC
             if (onErrorCallback != null)
                 newLayer.OnParseError.AddListener(onErrorCallback.Invoke);
 
-            
-            newLayer.SetURL(localFilePath, localFile.SourceUrl);
+            var fullPath = localFile.LocalFilePath;
+            var fileName = Path.GetFileName(fullPath);
+            var propertyData = newLayer.PropertyData as LayerURLPropertyData;
+
+            if (localFile.SourceUrl.StartsWith("http") || localFile.SourceUrl.StartsWith("https"))
```

Comment:
localFile.SourceUrl.StartsWith("https") is overbodig omdat dan ook localFile.SourceUrl.StartsWith("http") true is